pkgs/development/tools/refurb/default.nix

```
@@ -0,0 +1,68 @@
+{ lib
+, fetchFromGitHub
+, python3
```

This can be simplified by using.
```suggestion
, python3Packages
```